lib/rest-api.php

```diff
@@ -230,13 +230,10 @@ function gutenberg_add_permalink_template_to_posts( $response, $post, $request )
 		require_once ABSPATH . '/wp-admin/includes/post.php';
 	}
 
-	$sample_permalink = get_sample_permalink( $post->ID );
+	$sample_permalink = get_sample_permalink( $post->ID, $post->post_title, '' );
 
 	$response->data['permalink_template'] = $sample_permalink[0];
-
-	if ( 'draft' === $post->post_status && ! $post->post_name ) {
-		$response->data['draft_slug'] = $sample_permalink[1];
```

Comment:
🤔 Removing this is technically a breaking change. Since it was added recently and isn't documented, though, my feeling is that it's fine to leave as is.

Reply:
Yah, it's only in Gutenberg: there's not even a ticket for adding it to Core yet.